core/build-image.sh

```diff
@@ -5,6 +5,7 @@ cleanup_list=()
 trap 'rm -rf "${cleanup_list[@]}"' EXIT
 images=()
 repobase="${REPOBASE:-ghcr.io/nethserver}"
+redis_tag=7.2-rc-alpine
```

Comment:
I do not like using the rc version.
From the changelog:
```
Redis Release Candidate (RC) versions are early versions that are made available
for early adopters in the community to test them. We do not consider
them suitable for production environments.
```

Can we switch to 7.0?